FrontEnd/src/UserAuthentication/SignUpPage.jsx

```diff
@@ -79,6 +79,7 @@ function SignUpPage() {
         <input
           name="name"
           id="name"
+          placeholder="FirstName LastName"
```

Comment:
I'd rather provide an example than `FirstName LastName`:

```suggestion
          placeholder="ex. Jane Doe"
```